AppShell.xaml.cs

```diff
@@ -6,15 +6,18 @@ namespace HouseMovingAssistant;
 
 public partial class AppShell : Shell
 {
+    //TODO This seems not to be used 
```

Comment:
This is used. It is called when the application starts.

Reply:
Here I was referring to `viewModel`. It seems it's only assigned here, but it's never called/used otherwise